src/style.css

```diff
@@ -0,0 +1,129 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  box-sizing: border-box;
+  margin: 0 8px;
+  font-family: Avenir, sans-serif;
+}
+
+input {
+  width: 100%;
+  border: none;
+}
+
+textarea:focus,
+input:focus {
+  outline: none;
+}
+
+.form {
+  margin: 20px auto 0;
+}
+
+.form-big {
+  height: 70px;
+}
+
+.form-small {
+  height: 42px;
+}
+
+.label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.input-form {
+  font-family: Avenir, sans-serif;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 900;
+}
+
+.input-big {
+  padding-top: 23px;
+  padding-bottom: 23px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.img-big {
+  width: 19px;
+  height: 19px;
+}
+
+.input-small {
+  padding-top: 11px;
+  padding-bottom: 10px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.img-small {
+  width: 11px;
+  height: 11px;
+}
+
+.search-button {
+  position: absolute;
+  padding: 0;
+  border: none;
+  font: inherit;
+  color: inherit;
+  background-color: transparent;
+  cursor: pointer;
+}
+
+.search-button-big {
+  top: 25px;
+  left: 26px;
+}
```

Comment:
you have no medium state for search, so do not use `big`
left default and small
https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md point 2 of checklist